contracts/DelFiPrice.sol

```diff
@@ -9,17 +9,28 @@ import "./OpenOracleView.sol";
  * @author Compound Labs, Inc.
  */
 contract DelFiPrice is OpenOracleView {
+
     /**
      * @notice The event emitted when a price is written to storage
      */
     event Price(string symbol, uint64 price);
 
+    address anchor;
+    uint256 upperBoundAnchorRatio;
+    uint256 lowerBoundAnchorRatio;
```

Comment:
Prob want some natspecs on these, maybe as well change to the `///` form?

Reply:
done